kotlin/build.gradle.kts

```
@@ -22,8 +22,15 @@ testing {
         val testNg by registering(JvmTestSuite::class) {
             useTestNG()
 
-            group = "Verification"
-            description = "Runs the TestNG unit tests"
+
+            targets {
```

`armeria/kotlin/build.gradle.kts:25:1 Needless blank line(s) (no-consecutive-blank-lines)`

```suggestion
            targets {
```